pkg/cloudprovider/vsphereparavirtual/ippool/ippoolv1alpha1/ippoolv1alpha1.go

```diff
@@ -0,0 +1,160 @@
+package ippoolv1alpha1
```

Comment:
Maybe just rename the folder to v1alpha1?  `pkg/cloudprovider/vsphereparavirtual/v1alpha1/ippool`?